packages/app/lib/l10n/gen/app_localizations_es.dart

```diff
@@ -1,3 +1,5 @@
+// ignore: unused_import
```

Comment:
This file is generated, should it be out of lint checking instead?

Reply:
the lint ignore was generated too so I think this is normal for generated files 👍